internal/rukpak/api/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,151 @@
+	if in == nil {
+		return nil
+	}
+	out := new(Authorization)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *BundleSource) DeepCopyInto(out *BundleSource) {
+	*out = *in
+	if in.Image != nil {
+		in, out := &in.Image, &out.Image
+		*out = new(ImageSource)
+		**out = **in
+	}
+	if in.Git != nil {
+		in, out := &in.Git, &out.Git
+		*out = new(GitSource)
+		**out = **in
+	}
+	if in.ConfigMaps != nil {
+		in, out := &in.ConfigMaps, &out.ConfigMaps
+		*out = make([]ConfigMapSource, len(*in))
+		copy(*out, *in)
+	}
+	if in.HTTP != nil {
+		in, out := &in.HTTP, &out.HTTP
+		*out = new(HTTPSource)
+		**out = **in
+	}
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new BundleSource.
+func (in *BundleSource) DeepCopy() *BundleSource {
+	if in == nil {
+		return nil
+	}
+	out := new(BundleSource)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ConfigMapSource) DeepCopyInto(out *ConfigMapSource) {
+	*out = *in
+	out.ConfigMap = in.ConfigMap
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ConfigMapSource.
+func (in *ConfigMapSource) DeepCopy() *ConfigMapSource {
+	if in == nil {
+		return nil
+	}
+	out := new(ConfigMapSource)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *GitRef) DeepCopyInto(out *GitRef) {
+	*out = *in
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new GitRef.
+func (in *GitRef) DeepCopy() *GitRef {
+	if in == nil {
+		return nil
+	}
+	out := new(GitRef)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *GitSource) DeepCopyInto(out *GitSource) {
+	*out = *in
+	out.Ref = in.Ref
+	out.Auth = in.Auth
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new GitSource.
+func (in *GitSource) DeepCopy() *GitSource {
+	if in == nil {
+		return nil
+	}
+	out := new(GitSource)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *HTTPSource) DeepCopyInto(out *HTTPSource) {
+	*out = *in
+	out.Auth = in.Auth
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new HTTPSource.
+func (in *HTTPSource) DeepCopy() *HTTPSource {
+	if in == nil {
+		return nil
+	}
+	out := new(HTTPSource)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
We need not have BundleSource as a separate API which means as mentioned in the previous comment we can remove these deep copy methods.